source/codegen/metadata_mutation.py

```diff
@@ -214,6 +131,81 @@ def add_attribute_values_enums(enums, attribute_enums_by_type, group_name):
         add_enum(unmapped_enum_name, unmapped_values, enums, enum_value_prefix)
         add_enum(mapped_enum_name, mapped_values, enums, enum_value_prefix, is_mapped=True)
     
+
+AttributeReferencingParameter = namedtuple("AttributeReferencingParameter", ["attribute_group", "parameter"])
+
+class AttributeAccessorExpander:
```

Comment:
Formerly AttributeValueExpander.

Has significant changes.

You can see the move in a separate commit if you look at the history. But it's easier to just read it here as-is.